search_service/api/table.py

```diff
@@ -85,16 +83,8 @@ def __init__(self) -> None:
         super(SearchTableFieldAPI, self).__init__()
 
     @marshal_with(search_table_results)
-    def get(self, *, field_name: str,
-            field_value: str) -> Iterable[Any]:
-        """
-        Fetch search results based on query_term.
```

Comment:
One thing I notice is that the docstring is removed. Do you think we could keep the docstring as it is beneficial for development ? Or the concern is now it will have two copy of docstring (one in comment, one in swagger file)?

Reply:
^^ @swatro

Reply:
Yah I was hoping developers would rely on the swagger yml in order to understand the API signature. Instead of reading and updating the doc string, they should read and update the yml file. Thoughts? Is that too far away from the method signature.

Reply:
good point, but I think let's keep the docstring for now as I think it will be still good to see the doc without having a local instance/or check the swagger file running.  Hope it is ok for you.